atd-vze/src/Components/DataTable.js

```diff
@@ -201,6 +200,17 @@ const DataTable = ({
                                         data-1p-ignore
                                       />
                                     )}
+                                    {fieldUiType === "boolean" && (
+                                      <Input
+                                        autoFocus
+                                        defaultValue={fieldValue}
+                                        type="select"
+                                        onChange={e => handleInputChange(e)}
+                                      >
+                                        <option value={true}>TRUE</option>
```

Comment:
do we want the flags to say TRUE/FALSE or YES/NO? i went with true false cuz thats what we put for the person experiencing homelessness flag in the related records table

Reply:
I prefer Yes/No everywhere, including the PEH flag on the related records tables. I can open a separate issue if you prefer.

Reply:
I update to yes/no for both!